src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml

```diff
@@ -43,27 +58,13 @@ sylius_fixtures:
                             dhl_express:
                                 code: "dhl_express"
                                 name: "DHL Express"
+                                channels:
+                                    - "US Web Store"
```

Comment:
Yup, but then we should define channel code rather than base on generated value.